tests/integration/API/JSON/ResponseBuilders/FormsResponseBuilderTest.php

```diff
@@ -4,12 +4,62 @@
 
 use MailPoet\DI\ContainerWrapper;
 use MailPoet\Entities\FormEntity;
+use MailPoet\Models\StatisticsForms;
 use MailPoetVendor\Doctrine\ORM\EntityManager;
 
 class FormsResponseBuilderTest extends \MailPoetTest {
+  /** @var ContainerWrapper */
+  protected $container;
+
+  /** @var string */
+  protected $formName;
+
+  /** @var array */
+  protected $formBody;
+
+  /** @var array */
+  protected $formSettings;
+
+  public function _before() {
+    parent::_before();
+
+    $this->container = ContainerWrapper::getInstance();
+    $this->entityManager = $this->container->get(EntityManager::class);
+  }
+
   public function testItBuildsForm() {
-    $name = 'Form Builder Test';
-    $body = [
+    $form = $this->createForm('Form 1');
+
+    $responseBuilder = new FormsResponseBuilder();
+    $response = $responseBuilder->build($form);
+
+    expect($response['name'])->equals($this->formName);
+    expect($response['status'])->equals(FormEntity::STATUS_ENABLED);
+    expect($response['body']['name'])->equals($this->formBody['name']);
+    expect($response['body']['params']['label'])->equals($this->formBody['params']['label']);
+    expect($response['settings']['success_message'])->equals($this->formSettings['success_message']);
+  }
+
+  public function testItBuildsFormsForListing() {
+    $form1 = $this->createForm('Form 1');
+    $form2 = $this->createForm('Form 2');
+
+    $responseBuilder = new FormsResponseBuilder();
+    $response = $responseBuilder->buildForListing([$form1, $form2]);
+
+    expect($response)->count(2);
+    expect($response[0]['signups'])->equals(0);
+    expect($response[0]['segments'])->equals($this->formSettings['segments']);
+  }
+
+  public function _after() {
+    $this->truncateEntity(FormEntity::class);
+    StatisticsForms::deleteMany();
```

Comment:
Looks like we don't have a Doctrine entity for StatisticsForms yet. I'll create a Jira for that.

Reply:
Thanks